hztb-rest-api/src/main/java/com/svs/hztb/api/gcm/model/notification/MessageRequest.java

```diff
@@ -6,22 +6,22 @@
 import java.util.Map;
 
 public class MessageRequest {
-	private List<String> registrationIds;
+	private List<String> registration_ids;
 	private Map<String, String> data;
 
-	public List<String> getRegistrationIds() {
-		return registrationIds;
+	public List<String> getRegistration_ids() {
+		return registration_ids;
 	}
 
-	public void setRegistrationIds(List<String> registrationIds) {
-		this.registrationIds = registrationIds;
+	public void setRegistration_ids(List<String> registration_ids) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00100)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00117)